app/assets/javascripts/discourse/app/models/category.js

```diff
@@ -510,6 +510,11 @@ Category.reopenClass({
     return _.sortBy(data, category => {
       return category.get("read_restricted");
     });
+  },
+
+  extraBadgeIconRenderers: [],
```

Comment:
If these are only used in the helper I'd recommend moving them into there, and adding a `plugin-api` method to add one.

Reply:
Oh, I didnt think about adding the actual array to the PluginApi. Okay, great!